bundles/org.openhab.binding.siemenshvac/README.md

```diff
@@ -0,0 +1,111 @@
+baseUrl         | yes            |                | The address of the OZW672 devices
+userName        | yes            | Administrator  | The user name to log into the OZW672
+userPass        | yes            |                | The user password to log into the OZW672
+
+## Thing Configuration
+
+## Channels
+
+Channels are auto-discovered, you will find them on the RVS things.
+They are organized the same way as the LCD screen of your PAC device, by top level menu functionality, and sub-functionalities.
+Each channel is strongly typed, so for example, for heating mode, openHAB will provide you with a list of choices supported by the device.
+
+Channel                | Description                                              | Type          | Unit     | Security Access Level   |  ReadOnly | Advanced
+-----------------------|----------------------------------------------------------|---------------|----------|-------------------------|-----------|-------------------
+controlBoilerApproval  | Set Boiler Approval (`AUTO`, `OFF`, `ON`)                | String        |          |                         |  R/W      | true
+controlProgram         | Set Program (`OFF`, `NORMAL`, `WARMWATER`, `MANUAL`)     | String        |          |                         |  R/W      | true
+
+Channel Type ID        | Item Type                                | Description
+-----------------------|------------------------------------------|----------------------------------------------
+Numeric                | Number                                   | Handle basic numeric value
+String                 | String                                   | a String
+TimeOfDay              | TimeOfDay                                |
+Datetime               | Datetime                                 |
+RadioButton            | RadioButton                              |
+Scheduler              | Scheduler                                |
+Temperature            | Number                                   | Use to handle reading of  temperatur
+Setpoint               | Number                                   | Handle the setting of a temperature
+Regime                 | Number                                   | Enumeration for handling mode change
+   
+## Full Example
+
+Things file `.things`
+
+```java
+Bridge siemenshvac:ozw672:local "Ozw672" [ baseUrl="https://192.168.254.42/", userName="Administrator", userPassword="mypass"  ] 
+{
+    Thing RVS41_813_327 local "RVS41.813/327"  [  ]
+    {
+        Channels:
+            Type Setpoint:temperature                 "Temperature"  [ id="1726" ]
+            Type Regime:cc1                           "CC1"          [ id="1725" ]
+    
+    }
+```
+
+Items file `.items`
+
+```java
+String              Boiler_State_Pump_HWS       "HWS Pump State [%s]"                   { channel = "siemenshvac:RVS41_813_327:local:local:2237#2259_PpeChargeECS"              }       
```

Comment:
It's valid, but can change from installation to other installation.
In fact, there is three different ID that uis used.

In the following description : 2237 is the menuId, and 2259 is the item ID.
You can see them is the screenshoot I put above.

The is also the datapointId, what I call at some place the dptId, and what the thind in the channel description as "Id".
This is the important one, because we use it has the parameters in the read/write api operations.

Type Setpoint:temperature                 "Temperature"  [ id="1726" ]

So the first one is not really important : "siemenshvac:RVS41_813_327:local:local:2237#2259_PpeChargeECS" ,you just need to match what is list in the autodiscovery, or in the Channel definition if you do it manually.

Reply:
fix the channel sample to match the last modification on the UidUtils.sanetizeId